### DISCUSSION TITLE
com.apple.developer.carplay-fueling

### DISCUSSION THREAD

**Discussion by redFrenesi** [oguzhnatly/flutter_carplay#45] [Q&A]
Dont work if i have the com.apple.developer.carplay-fueling CarPlay Entitlements, what can i do? helpMe please

**Comment by oguzhnatly** [comment]
The `com.apple.developer.carplay-fueling` entitlement requires specific templates that Apple allows for fueling apps (like `CPPointOfInterestTemplate`). Make sure you're only using templates permitted for your entitlement category. Check Apple's [CarPlay entitlements documentation](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_carplay-fueling) for the allowed template types. If you're using templates outside your category, CarPlay will fail silently.
